From 3e6ca96e102cc8ccb7fc68911b5e9a7a3a54df3d Mon Sep 17 00:00:00 2001
From: "Wuschel-Wuschel/skin.osmc"
 <36623079+Wuschel-Wuschel@users.noreply.github.com>
Date: Thu, 5 Jul 2018 12:05:44 +0200
Subject: [PATCH] Swap temp/wind in weather

MyWeather.xml:
- swap position of current temperature and wind information
---
 16x9/MyWeather.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/16x9/MyWeather.xml b/16x9/MyWeather.xml
index 87f75d0..d98077c 100755
--- a/16x9/MyWeather.xml
+++ b/16x9/MyWeather.xml
@@ -209,7 +209,7 @@
 					<height>45</height>
 					<font>Font33</font>
 					<textcolor>TextColor2</textcolor>
-					<label>$LOCALIZE[383]: $INFO[Window(Weather).Property(Current.Wind)]</label>
+					<label>$LOCALIZE[1375]: $INFO[Window(Weather).Property(Current.Temperature)]$INFO[System.TemperatureUnits] / $LOCALIZE[402]: $INFO[Window(Weather).Property(Current.FeelsLike)]$INFO[System.TemperatureUnits]</label>
 				</control>
 				<control type="label">
 					<left>0</left>
@@ -218,7 +218,7 @@
 					<height>45</height>
 					<font>Font30</font>
 					<textcolor>TextColor2</textcolor>
-					<label>$LOCALIZE[1375]: $INFO[Window(Weather).Property(Current.Temperature)]$INFO[System.TemperatureUnits] / $LOCALIZE[402]: $INFO[Window(Weather).Property(Current.FeelsLike)]$INFO[System.TemperatureUnits]</label>
+					<label>$LOCALIZE[383]: $INFO[Window(Weather).Property(Current.Wind)]</label>
 				</control>
 				<control type="label">
 					<left>600</left>
